problems/strings.js

```diff
@@ -1,38 +1,53 @@
-/**
+/**DONE
  * Returns the first character of a string passed in.
  * @param {string} - str
  * @returns {string} - first char
  */
-function firstChar() {}
+function firstChar(str) {
+  str = str.slice(0, 1);
```

Comment:
We want to try not change the value of the actual parameter so you should assign st.slice to a new variable